.storybook/styles.css

```diff
@@ -7,3 +8,8 @@
 .leaflet-container {
   position: absolute !important;
 }
+@media only screen and (min-width: 768px) {
+ * {
+  min-width: 0
+ }
+}
```

Comment:
What does this fix actually do?

Reply:
That was for fixing width issues on footer component in storybook. I'll remove it becouse we dont need it